pkg/client/client_test.go

```diff
@@ -192,16 +234,34 @@ var _ = Describe("Client", func() {
 			close(done)
 		})
 
-		It("should fail if the object does not pass server-side validation", func() {
-			// TODO(seans3): implement these
-		})
+		It("should fail if the object does not pass server-side validation", func(done Done) {
+			cl, err := client.New(cfg, client.Options{})
+			Expect(err).NotTo(HaveOccurred())
+			Expect(cl).NotTo(BeNil())
+
+			By("creating the pod, since required field Containers is empty")
+			err = cl.Create(context.TODO(), pod)
+			Expect(err).To(HaveOccurred())
```

Comment:
Would you also check the error message to make sure it is actually the expected message and not something unrelated?

Reply:
Done